ga4gh/server/repo/rnaseq2ga.py

```diff
@@ -157,6 +190,7 @@ def writeExpression(self, rnaQuantificationId, quantfilename,
                               rawCount, score, units, confidenceLow,
                               confidenceHi)
                 self._db.addExpression(datafields)
+                expressionId += 1
```

Comment:
Cool! Do you think this same idea will work for adding expressions in parallel (ignoring sqlite)?